lib/qesapdeployment.pm

```diff
@@ -413,4 +414,59 @@ sub qesap_ansible_cmd {
     assert_script_run($ansible_cmd);
 }
 
+
+=head3 qesap_ansible_script_output
+
+    Use Ansible to run a command remotely and get the stdout.
+    Command is executed with elevated privileges
+
+    qesap_ansible_script_output(cmd => 'crm status', provider => 'aws', host => 'vmhana01');
+
+    It uses playbook data/sles4sap/script_output.yaml
+
+    1. ansible-playbook run the playbook, remote cmd ar paramenter
```

Comment:
ar? Not sure whether it should be `remote cmd or parameter` or `remote cmd and parameter`. There's also a typo in parame**n**ter.